.vib/drupal/cypress/cypress/integration/drupal_spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows creating an article with an image', () => {
+  cy.login();
+  cy.visit('/node/add/article');
+  cy.fixture('articles').then((article) => {
+    cy.get('.text-full[name*=title]').type(`${article.newArticle.title} ${random}`);
+    cy.get('iframe.cke_wysiwyg_frame').iframe().find('p').type(`${article.newArticle.text}`);
+    cy.get('[type=file]').scrollIntoView();
+    cy.get('[type=file]').selectFile('cypress/fixtures/images/test_image.jpeg', { force: true });
+    cy.get('img').should('have.attr', 'src').and('contains', 'test_image');
+    cy.get('input[id*="alt"]').type(`${article.newArticle.altText}`);
```

Comment:
Checking the app, this field only appears when the image is fully uploaded so we can avoid checking for `test_image` above as this one does basically the same.

Reply:
I think the test fails without this check, but I will double check.

Reply:
You were right, I removed it